api/api-reference.md

```diff
@@ -0,0 +1,849 @@
+---
```

Comment:
General commenst:
- I don't see why all headers, pseudo headers, and bullets on this page should have every word capitalized? 
- Variables?